extras/rya.geoindexing/src/main/java/org/apache/rya/indexing/OptionalConfigUtils.java

```diff
@@ -83,11 +90,14 @@ public static void setIndexers(final RdfCloudTripleStoreConfiguration conf) {
 
         boolean useFilterIndex = false;
         ConfigUtils.setIndexers(conf);
-        for (final String index : conf.getStrings(AccumuloRdfConfiguration.CONF_ADDITIONAL_INDEXERS)){
-            indexList.add(index);
-        }
-        for (final String optimizer : conf.getStrings(RdfCloudTripleStoreConfiguration.CONF_OPTIMIZERS)){
-            optimizers.add(optimizer);
+        final String[] existingIndexers = conf.getStrings(AccumuloRdfConfiguration.CONF_ADDITIONAL_INDEXERS);
```

Comment:
this was an issue i discovered while writing tests before there was a config field for this indexer